frontend/components/templates/layouts/TopLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import Navbar from '@/components/organisms/Navbar'
+
+type Props = {
+    children: JSX.Element
+}
+
+const TopLayout = ({ children }: Props): JSX.Element => {
+    return (
+        <div className="relative h-screen">
+            <div className="fixed z-20 h-14 w-full border-b-2 border-neutral-200 drop-shadow-[0_2px_2px_rgba(0,0,0,0.05)]">
```

Comment:
Drop shadow is repetitive. You can create a config for this.
https://tailwindcss.com/docs/drop-shadow#customizing-your-theme

Reply:
This has been applied